x-pack/plugins/security_solution/public/management/pages/policy/view/ingest_manager_integration/endpoint_package_custom_extension/components/exception_items_summary.tsx

```diff
@@ -36,46 +37,89 @@ const SUMMARY_LABELS: Readonly<{ [key in keyof GetExceptionSummaryResponse]: str
-        return (
-          <EuiFlexItem key={stat}>
-            <SummaryStat
-              value={stats?.[stat] ?? 0}
-              color={stat === 'total' ? 'primary' : 'default'}
-              key={stat}
-            >
-              {SUMMARY_LABELS[stat]}
-            </SummaryStat>
-          </EuiFlexItem>
-        );
-      })}
-    </EuiFlexGroup>
-  );
-});
+export const ExceptionItemsSummary = memo<ExceptionItemsSummaryProps>(
+  ({ stats, small = false }) => {
+    const getItem = useCallback(
+      (stat: keyof GetExceptionSummaryResponse) => (
+        <EuiFlexItem key={stat}>
+          <SummaryStat
+            value={stats?.[stat] ?? 0}
+            color={stat === 'total' ? 'primary' : 'default'}
+            key={stat}
+            small={small}
+          >
+            {SUMMARY_LABELS[stat]}
+          </SummaryStat>
+        </EuiFlexItem>
+      ),
+      [stats, small]
+    );
+
+    return (
+      <EuiFlexGroup
+        alignItems="center"
+        justifyContent={small ? 'flexStart' : 'spaceAround'}
+        gutterSize={small ? 's' : 'l'}
+      >
+        {SUMMARY_KEYS.map((stat) => getItem(stat))}
+      </EuiFlexGroup>
+    );
+  }
+);
 
 ExceptionItemsSummary.displayName = 'ExceptionItemsSummary';
 
-const SummaryStat: FC<{ value: number; color?: EuiBadgeProps['color'] }> = memo(
-  ({ children, value, color, ...commonProps }) => {
+const SummaryStat: FC<{ value: number; color?: EuiBadgeProps['color']; small?: boolean }> = memo(
+  ({ children, value, color, small = false, ...commonProps }) => {
     return (
-      <EuiText className="eui-displayInlineBlock" size="s">
-        <EuiFlexGroup justifyContent="center" direction="row" alignItems="center">
-          <EuiFlexItem grow={false} style={color === 'primary' ? CSS_BOLD : undefined}>
-            {children}
-          </EuiFlexItem>
-          <EuiFlexItem grow={false}>
-            <EuiBadge color={color}>{value}</EuiBadge>
-          </EuiFlexItem>
-        </EuiFlexGroup>
+      <EuiText className="eui-displayInlineBlock" size={small ? 'xs' : 's'}>
+        <StyledEuiFlexGroup
+          justifyContent={small ? 'flexStart' : 'center'}
+          direction="row"
+          alignItems="center"
+          gutterSize={small ? 'xs' : 'l'}
+          small={small}
+        >
+          {small ? (
+            <>
+              <EuiFlexItem grow={false}>
+                <EuiBadge color={color}>{value}</EuiBadge>
+              </EuiFlexItem>
+              <EuiFlexItem grow={false} style={color === 'primary' ? CSS_BOLD : undefined}>
+                {children}
+              </EuiFlexItem>
+            </>
+          ) : (
+            <>
+              <EuiFlexItem grow={false} style={color === 'primary' ? CSS_BOLD : undefined}>
+                {children}
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiBadge color={color}>{value}</EuiBadge>
+              </EuiFlexItem>
+            </>
```

Comment:
This can be done using flex properties no? Just align with a `row` or `rowReverse` based on `isSmall`?

Reply:
totally right, thanks!